build/Dockerfile.ubuntu

```diff
@@ -57,18 +55,28 @@ RUN apt-get install -y apt-utils \
     curl \
     connect-proxy \
     coreutils \
-    sudo && \
-    apt-get -y clean all
+    vim \
+    sudo
 
-# Installing all PYTHON packages
-RUN pip install --upgrade pip && \
-    pip install grpcio \
-    ovspy \
+RUN if [ "$BASE_IMG" = "ubuntu:18.04" ] ; then \
+       apt-get -y install python-pip; \
+    else \
+       apt-get -y install pip; \
+    fi
+
+RUN apt-get -y clean all
```

Comment:
When running `apt-get -y clean all` as a separate step in the Dockerfile, it would create a new layer in the container image with [whiteout files](https://github.com/opencontainers/image-spec/blob/main/layer.md#whiteouts). As a result all deleted files will remain in the previous layer.

Reply:
This is a great find, and I agree with fixing it before this is merged.